src/backend/web/routes/index.js

```diff
@@ -17,7 +17,6 @@ const user = require('./user');
 const query = require('./query');
 
 const router = express.Router();
-
```

Comment:
Watch out for spurious whitespace changes like this, as it churns git history unnecessarily.